modular_skyrat/modules/customization/modules/client/preferences.dm

```diff
@@ -192,7 +193,7 @@ GLOBAL_LIST_INIT(food, list(
 	/// If we have persistent scars enabled
 	var/persistent_scars = TRUE
 	///If we want to broadcast deadchat connect/disconnect messages
-	var/broadcast_login_logout = FALSE
+	var/broadcast_login_logout = TRUE
```

Comment:
I mean, does that need to be on by default? Not that it matters, I'm just curious what the thoughts are behind it.

Reply:
TG standard reset